packages/discord.js/src/structures/BaseInteraction.js

```diff
@@ -107,6 +107,21 @@ class BaseInteraction extends Base {
       (coll, entitlement) => coll.set(entitlement.id, this.client.application.entitlements._add(entitlement)),
       new Collection(),
     );
+
+    /* eslint-disable max-len */
+    /**
+     * Mapping of installation contexts that the interaction was authorized for the related user or guild ids
+     * @type {APIAuthorizingIntegrationOwnersMap}
+     * @see {@link https://discord.com/developers/docs/interactions/receiving-and-responding#interaction-object-authorizing-integration-owners-object}
+     */
+    this.authorizingIntegrationOwners = data.authorizing_integration_owners;
+    /* eslint-enable max-len */
+
+    /**
+     * Context where the interaction was triggered from
+     * @type {?InteractionContextType}
+     */
+    this.context = data.context ?? null;
```

Comment:
The initialization of these properties should be removed from the CommandInteraction constructor if BaseInteraction is now handling the initialization of them.